### DISCUSSION TITLE
Delete old data

### DISCUSSION THREAD

**Discussion by fbattistin** [pfelk/pfelk#459] [Q&A]
Hello,
I'm not an Elasticsearch admin, so I need your help: I would like to delete data older than 60 days from elasticsearch; What's the best way to do it?

Thank you in advance.

**Comment by a3ilson** [comment]
@fbattistin - I would recommend configuring an ILM

https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html
